<ui version="4.0" >
 <class>PreferencePageFlowMeter</class>
 <widget class="QWidget" name="PreferencePageFlowMeter" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>PreferencePageFlowMeter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QGroupBox" name="_fuelSelectionGroupBox" >
     <property name="title" >
      <string>Fuel selection</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QRadioButton" name="_petrolRadioButton" >
        <property name="text" >
         <string>Petrol</string>
        </property>
       </widget>
      </item>
      <item rowspan="2" row="0" column="1" >
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>286</width>
          <height>39</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" >
       <widget class="QRadioButton" name="_gasRadioButton" >
        <property name="text" >
         <string>Gas</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="_fuelOptionsGroupBox" >
     <property name="title" >
      <string/>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>59</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
